src/main/java/org/mockito/internal/creation/bytebuddy/MockMethodAdvice.java

```diff
@@ -45,6 +47,13 @@
     public MockMethodAdvice(WeakConcurrentMap<Object, MockMethodInterceptor> interceptors, String identifier) {
         this.interceptors = interceptors;
         this.identifier = identifier;
+        try {
+            Field field = WeakConcurrentMap.class.getDeclaredField("target");
+            field.setAccessible(true);
+            backingMap = field.get(interceptors);
+        } catch (Exception e) {
+            throw new MockitoException("Could not resolve backing map from concurrent hash map", e);
```

Comment:
Can we make the message more actionable? Is there anything the user can do? File a ticket?

Reply:
It is my library, I can add a method to get hold of the object. But I think we are safe here as we shade the code, no real danger here.

Reply:
Nevermind, we do of course inline it and can therefore change the code.